docs/api/Module.md

```diff
@@ -29,10 +29,10 @@ If a native module named `id` exists, load it and return.
 
 `require` function searches for modules in the following order:
 
-1. Current working directory.
-2. `iotjs_modules` folder under current working directory.
-3. `$HOME/iotjs_modules`
-4. `$IOTJS_PATH/iotjs_modules`
+1. `$NODE_PRIORITIZED_PATH/node_modules` if present.
+1. `node_modules` folder under current working directory.
+2. `$NODE_PATH/node_modules`.
```

Comment:
Actually, if a `NODE_PATH` which is current working directory, relative js file can be loaded without leading `./`.

e.g. with following directory structure:

```
- /home/workspace
|- lib
|    |- foobar.js
|- index.js
```

and set `NODE_PATH` to `/home/workspace`,

and with following content in `index.js`, it works:

```js
require('lib/foobar')
```

So it should be `2. $NODE_PATH`

Reply:
The above example doesn't work at IoT.js and ShadowNode. The search path for NODE_PATH should be `$NODE_PATH/node_modules`.

Reply:
It works in Node.js, should we support it too?